plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.xml

```diff
@@ -54,5 +55,38 @@
       markerType="com.google.cloud.tools.eclipse.appengine.validation.blacklistMarker">
     </markerResolutionGenerator>
   </extension>
+  
+    <extension 
+      name="%web.xml"
+      point="org.eclipse.wst.validation.validatorV2" 
+      id="webXmlValidator">
+    <validator
+    	class="com.google.cloud.tools.eclipse.appengine.validation.WebXmlValidator"
+    	markerId="com.google.cloud.tools.eclipse.appengine.validation.servletMarker">
+      <include>
```

Comment:
We can use exclude with facet rules to keep this validator from firing on flex projects, perhaps in the next PR.